Add more details about Microsoft Teams (#3067)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeServer/recipients/room-mailboxes.md

If you're planning to have more to have hundreds of rooms, use multiple room lists to help you organize your rooms. If your company has several buildings with rooms that can be booked for meetings, it might help to create room lists for each building. Room lists are specially marked distribution groups that you can use the same way you use distribution groups. However, you can only create room lists using the Exchange Management Shell.

> [!NOTE]

> > Although there is no hard limit to the number of rooms you can have in a Room List, the maximum number of rooms that can be returned in request for a Room List is 100 (including when you're using Microsoft Teams to create a meeting invite). A possible workaround would be to further break down your rooms into smaller lists.

Reviewer: ```suggestion
> Although there is no hard limit to the number of rooms you can have in a Room List, the maximum number of rooms that can be returned in a request for a Room List is 100 (including when you're using Microsoft Teams to create a meeting invite). A possible workaround would be to further break down your rooms into smaller lists.
```

> [!NOTE]

> > When using Microsoft Teams, the list of rooms could be shown in any particular order and not necessarily in alphabetical order as the internal process to get the list does not sort the output.

Reviewer 2: ```suggestion
> When using Microsoft Teams, the list of rooms displays in random order. The internal process to get the list doesn't sort the output, so the list may not be in alphabetical order.
```
